OpenSim/Sandbox/ExampleHopperDevice/README.md

```diff
@@ -0,0 +1,29 @@
+Example: *Hopper Device* 
+========================
+
+This example demonstrates some of the new features of the OpenSim 4.0 API.
+The Component architecture allows us to join sub-assemblies to form larger Models, with information flowing between Components via Inputs, Outputs, and Connectors.
+For more information, please refer to the [OpenSim doxygen](http://doxygen.opensim.community/) for the Component class.
+
+This interactive example consists of three steps:
+
+| []()                                                                   | []()
```

Comment:
Also, interestingly, it is similar to the notation for lambdas in C++: `[](){}`.